staging/src/k8s.io/api/resource/v1alpha3/types.go

```diff
@@ -1021,7 +1021,7 @@ type ResourceClaimTemplate struct {
 
 // ResourceClaimTemplateSpec contains the metadata and fields for a ResourceClaim.
 type ResourceClaimTemplateSpec struct {
-	// ObjectMeta may contain labels and annotations that will be copied into the PVC
+	// ObjectMeta may contain labels and annotations that will be copied into the ResourceClaim
```

Comment:
Thanks. Please also fix in `pkg/apis/resource/types.go`, then we can try to get this approved.

Reply:
Done